pkg/bisect/bisect.go

```diff
@@ -710,6 +715,13 @@ func (env *env) processResults(current *vcs.Commit, results []instance.EnvTestRe
 	return
 }
 
+func (env *env) isTransientError(rep *report.Report) bool {
```

Comment:
Isn't this overly specific towards SYZFATAL? I assume it's quite common to run into two different bugs at the same time. There are also certain ranges of the kernel history that tend to crash a lot with out much correlation to the reproducer. Should we consider being even more conservative and ignore all errors that don't resemble the expected crash?

Reply:
I was trying to stay on the safe side here. Unexpected SYZFATAL was a very obvious example of a crash we should not pursue (+ I saw some of my local bisections take the wrong turn because of that).

It would surely be best if we could reliably determine that one error resembles another. But can we really do that in all cases? When we bisect over large ranges, code can change significantly in between, and so can crash titles.

Reply:
Fair. Maybe we should set the flaky flag in case the title isn't consistent?

Reply:
Just so there is some indicator on the dashboard that this bisection might have gone off the rails.

Reply:
Interesting.

Though we should probably avoid the doubling of VM instances in this case, as we do for really flaky bisections (when there are only a few crashes per run). It won't help much in this case.